SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/StaticSubobjectsTest/StaticSubobjectTestActor.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "StaticSubobjectTestActor.h"
+#include "Net/UnrealNetwork.h"
+
+AStaticSubobjectTestActor::AStaticSubobjectTestActor()
+{
+	TestIntProperty = 0;
+	bNetLoadOnClient = true;
+	bNetLoadOnNonAuthServer = true;
+	bReplicates = true;
+
+	TestStaticComponent1 = CreateDefaultSubobject<USceneComponent>(TEXT("ToRemoveComponent1"));
+	TestStaticComponent1->SetupAttachment(this->GetRootComponent(), TEXT("ToRemoveComp1Socket"));
+	TestStaticComponent1->SetIsReplicated(true);
+
+	TestStaticComponent2 = CreateDefaultSubobject<USceneComponent>(TEXT("ToRemoveComponent2"));
+	TestStaticComponent2->SetupAttachment(this->GetRootComponent(), TEXT("ToRemoveComp2Socket"));
+	TestStaticComponent2->SetIsReplicated(true);
```

Comment:
Sockets aren't required, and don't do anything if the RootComponent doesn't have these sockets.

```suggestion
	TestStaticComponent1 = CreateDefaultSubobject<USceneComponent>(TEXT("ToRemoveComponent1"));
	TestStaticComponent1->SetupAttachment(RootComponent);
	TestStaticComponent1->SetIsReplicated(true);

	TestStaticComponent2 = CreateDefaultSubobject<USceneComponent>(TEXT("ToRemoveComponent2"));
	TestStaticComponent2->SetupAttachment(RootComponent);
	TestStaticComponent2->SetIsReplicated(true);
```